CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ This is the log of notable changes to EAS CLI and related packages.
 
 ### 🧹 Chores
 
+- Make sure that all links in messages are clickable, if used terminal supports that. ([#1591](https://github.com/expo/eas-cli/pull/1591) by [@Simek](https://github.com/Simek))
```

Comment:
Maybe this is me but this sentence doesn't make sense.

Reply:
Do you have any suggestions how to improve it? Which part seems off for you, or is it the whole sentence?

> Ensure that all URLs in messages are clickable, if used terminal supports mouse interaction. 

Does the entry above makes more sense?

Reply:
"Make all URLs in logs clickable in terminals supporting hyperlinks."?

Reply:
Want to suggest that change, so we can update it from web UI?

Reply:
```suggestion
- Make all URLs in logs clickable in terminals supporting hyperlinks. ([#1591](https://github.com/expo/eas-cli/pull/1591) by [@Simek](https://github.com/Simek))
```